Objects/typeobject.c

```diff
@@ -116,16 +116,10 @@ type_from_ref(PyObject *ref)
 
 /* helpers for for static builtin types */
 
-static inline int
-static_builtin_index_is_set(PyTypeObject *self)
-{
-    return self->tp_subclasses != NULL;
-}
```

Comment:
This function, and its similar neighbors, let us avoid leaking implementation details out to other parts of the file, which I think is important, so I'd rather we wrap the function in an ifdef:

```C
#ifndef NDEBUG
static inline int
static_builtin_index_is_set(PyTypeObject *self)
{
    return self->tp_subclasses != NULL;
}
#endif
```

Reply:
I did actually consider this :) I'll update the PR right away.

Reply:
Addressed in b67d7bd127409e50594bf2fb17c69980743c282e